src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs

```diff
@@ -13,44 +13,44 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
-using System;
+using System;
 using System.Collections.Generic;
 
 namespace OpenTelemetry.Exporter.Jaeger
 {
     public class JaegerExporterOptions
     {
-        internal const string DefaultServiceName = "Open Telemetry Exporter";
+        internal const string DefaultServiceName = "OpenTelemetry Exporter";
 
-        internal const int DefaultMaxPacketSize = 65000;
-
-        /// <summary>
-        /// Gets or sets the name of the service reporting telemetry. Default value: Open Telemetry Exporter.
+        internal const int DefaultMaxPacketSize = 65000;
```

Comment:
This is a bit surprising, the current code actually has a mixture of CRLF and LF line endings in the same file.

![image](https://user-images.githubusercontent.com/17327289/91111136-8f97ce80-e634-11ea-89ac-326fbd1b55c4.png)

@cijothomas we probably need to have `.gitattribute` or a CI file to enforce this.

Reply:
I will push soon so we can test :)